app/src/components/inputs/inputRadio/inputRadio.jsx

```diff
@@ -59,6 +60,7 @@ export const InputRadio = ({
         checked: value === ownValue,
         'at-top': circleAtTop,
         [`mode-${mode}`]: mode,
+        [`${size}-toggler`]: size,
```

Comment:
Please reverse the name structure, as the first part should be common class name, as the second should be its variant, f.e. `toggler-${size}`.